### PR TITLE
make module_name to file_name be case sensitive

### PR DESCRIPTION
**Pull request by rbjorklin**
* Imported logger
* Raise SphinxError instead of exit(1)

### CODE REVIEW

**Comment by rbjorklin** [comment]
I tried running `sphinxcontrib-ocaml` on the `utop` project and have run in to a few problems. There might be more PRs coming if I can find the time :)

File: sphinxcontrib/ocaml/autodocumenters.py [hunk 1]
```diff
@@ -156,19 +161,17 @@ class ModuleDocumenter(sphinx.ext.autodoc.Documenter):
 
     def generate(self, more_content=None, real_modname=None, check_module=False, all_members=False):
         if self.env.config.ocaml_source_directories is None:
-            # @todo Use Sphinx's warnings and errors infrastructure
-            print("ERROR: please set ocaml_source_directories in conf.py")
-            exit(1)
+            raise SphinxError('Please set ocaml_source_directories in conf.py')
```
Comment (jacquev6) [inline sphinxcontrib/ocaml/autodocumenters.py:164]: Thanks!

File: sphinxcontrib/ocaml/autodocumenters.py [hunk 2]
```diff
@@ -156,19 +161,17 @@ class ModuleDocumenter(sphinx.ext.autodoc.Documenter):
...
 
         module_name = self.directive.arguments[0]
         for d in self.env.config.ocaml_source_directories:
-            interface_file_name = "{}/{}.mli".format(d, module_name)
-            if os.path.isfile(interface_file_name):
+            file_name = [f for f in os.listdir(d) if f.lower() == "{}.mli".format(module_name.lower())][0]
```
Comment (jacquev6) [inline sphinxcontrib/ocaml/autodocumenters.py:168]: Agreed, I was using this tool on my projects where I match the case of file names and modules, but this doesn't seem to be a widespread convention in the OCaml community.

**Comment by jacquev6** [comment]
@rbjorklin thank you for your contributions!

Would you like push access to the repo ?

Please let me know when you'd like a new release of the OPAM and PyPI packages.

**Comment by rbjorklin** [comment]
You're welcome!

That would be nice for small fixes but I'm probably still going to want to create PRs and have them reviewed before merging :)

Will do!

**Comment by jacquev6** [comment]
I've just invited you as a collaborator to this repo. Don't hesitate to push small fixes; I'll keep a eye on what you do. And when you feel it's better, you can still open PRs for me to review.

**Pull request merged**